upcase_booklist_app/templates/single_book.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='css/main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+    </head>
+
+    <body>
+        <div class="page-header">
+            <p class="nav-item"><a href="{{ url_for('book_list') }}">Books</a></p>
+        </div>
+                
+
+        <div class="single-book">
+            {% for category, details in single_book_details.items() %}
```

Comment:
I'd recommend job passing along the variable `book` instead of `single_book_details` to this template, because that's really what it is. And when you start using a database, each record, along with all it's details, will be called a `book`, so might as well get practice with it now.